.baseline/checkstyle/checkstyle.xml

```diff
@@ -354,7 +348,7 @@
             <property name="tagOrder" value="@param, @return, @throws, @deprecated"/>
             <property name="target" value="CLASS_DEF, INTERFACE_DEF, ENUM_DEF, METHOD_DEF, CTOR_DEF, VARIABLE_DEF"/>
         </module>
-        <!-- <module name="CyclomaticComplexity"/> --> <!-- Java Coding Guidelines: Reduce Cyclomatic Complexity -->
+        <module name="CyclomaticComplexity"/> <!-- Java Coding Guidelines: Reduce Cyclomatic Complexity -->
```

Comment:
Hmm, this seems a bit worrying to me (I'd imagine we would fail this check _horribly_), though I guess if the tests pass it should be okay.